### DISCUSSION TITLE
Get nested type from OASOutput

### DISCUSSION THREAD

**Discussion by Ashraf-Ali-aa** [ardatan/feTS#1002] [Q&A] [answered]
Hi, I was wondering if there was an elegant way of getting the nested type from the response an example I want to get the type interface for the address.

example:

```
{
	userid: 1,
	user: {
		name: 'test',
		address: {
			postcode: 'test'
		}

	}
}
```

```
type UserByIdResponse = OASOutput<NormalizeOAS<typeof oas>, '/user/:id', 'POST', 200>

```

**Comment by ardatan** [comment, marked as answer]
`type Address = OASOutput<NormalizeOAS<typeof oas>, '/user/:id', 'POST', 200>['user']['address']`
